app/experimenter/experiments/tests/test_forms.py

```diff
@@ -1047,17 +1047,31 @@ def test_sets_bugzilla_id_when_draft_becomes_review(self):
             self.user.id, experiment.id
         )
 
-    def test_adds_bugzilla_comment_when_review_becomes_ship(self):
+    def test_adds_bugzilla_comment_and_normandy_slug_when_becomes_ship(self):
         experiment = ExperimentFactory.create_with_status(
-            Experiment.STATUS_REVIEW, bugzilla_id="12345"
+            target_status=Experiment.STATUS_REVIEW,
+            type=Experiment.TYPE_PREF,
+            name="Experiment Name",
+            slug="experiment-slug",
+            firefox_version="57.0",
+            firefox_channel=Experiment.CHANNEL_NIGHTLY,
+            bugzilla_id="12345",
         )
+        self.assertEqual(experiment.normandy_slug, None)
+
         form = ExperimentStatusForm(
             request=self.request,
             data={"status": experiment.STATUS_SHIP},
             instance=experiment,
         )
+
         self.assertTrue(form.is_valid())
         experiment = form.save()
+
+        self.assertEqual(
+            experiment.normandy_slug,
+            "pref-experiment-slug-Nightly-57.0-bug-12345",
```

Comment:
This is fine, but slugs *should* be all lower case.

Reply:
Good call.